src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.nav {
```

Comment:
if you have block nav and block header please create a separate file for every bem block and import them to style.css